yapapi/strategy.py

```diff
@@ -71,18 +77,22 @@ def __str__(self):
 class MarketStrategy(DemandDecorator, abc.ABC):
     """Abstract market strategy."""
 
-    acceptable_prop_value_ranges: Dict[str, PropValueRange]
+    acceptable_prop_value_ranges_overrides: Dict[str, PropValueRange]
+    __acceptable_prop_value_ranges: Dict[str, PropValueRange]
+
+    @property
+    def acceptable_prop_value_ranges(self) -> Dict[str, PropValueRange]:
+        if not hasattr(self, "_acceptable_prop_value_ranges"):
+            # initialize with the overrides
+            self.__acceptable_prop_value_ranges = getattr(
+                self, "acceptable_prop_value_range_overrides", {}
```

Comment:
I don't understand this. When do we have `acceptable_prop_value_range_overrides` set, and when not?

ALSO: we have
`acceptable_prop_value_range_overrides` here, and
`acceptable_prop_value_ranges_overrides` as the class slot

Reply:
will fix in a minute